.travis.yml

```diff
@@ -1,10 +1,20 @@
 language: node_js
 node_js:
-  - "4"
-  - "6"
-  - "7"
+- '4'
+- '6'
+- '7'
 cache:
   directories:
-    - node_modules
+  - node_modules
 script:
-  - grunt
+- grunt
+deploy:
+  skip_cleanup: true
+  provider: npm
+  email: i.am.an.eternity@gmail.com
+  api_key:
+    secure: HHoNHOaayqMN03HfrmDqZoc1eSWK1yoo9UMPqpb0/ADMlWY7rliEv5mBwzBVIwtVnE52qgcpyoxQM9fA/6ZhTdfQZnGxGQaGYcSizTP7cBGM+KHE3tfJw/R3Kw33eRrfDqMQI/2FLmsqfj3FyYUfFcTwCnxXw2thOBiGmxHauW3UFr2U0zrNUM7U3ZI18ugsT2mpUzd9UwwFUogMPcKS+TzTJjmvuLPqp9XRFYAwglL4D/CrOcgLefcikhYVS+e+Pz/5ePVGhvNHtijZ+8JTR1lkpV70cr13wxP3Uhzq8otmKaLgeWZLYXvWI1+sfIGtfC4f0Q52kvdR4VQ8KojQsLXUr1ypkn4tz2m+VgzDqX2eeM+Dagey2/wxCRiAVAmpM8VpPB+mbbHAkWKq8ey/6ZGraJAWj1nnpzWSpYaA4tCKBJa2k+erKLgKSYXucSrNdp7/dvS1kStzelMF2Cc1ySRHDXnleDYlVCiLfDuUqyxmiOk5tr3USUq99dw7BFL9LR92K+H+2Ws8vtrMfgOVCy1AgtE67aFRmkccrEkAv5XNzZt/dbb4BuBE6OYntUA5kEhPoFqNXuEMO/D3Ytbw+0s0Mk1O+frxtdC/cULplqLVqXKqABwVb4ZYfMGLJ0N3ytjNvERLG8f3kN586gGzaM7vqf9gBCWv1tJrkwUxwSc=
+  on:
+    branch: master
+    tags: true
+    node: '6'
```

Comment:
Tip: You can add Node 8 instead of 7 and deploy on Node 8 in order to enable the [SHA-512 integrity check](https://docs.npmjs.com/cli/publish). Alternatively, you may try to update `npm` to v5 on some other Node (I'd recommend to do this in `before_deploy` so that compatibility with old `npm` versions is tested). The only issue with this would be the `prepublish` script in `package.json`, which is deprecated in favor of `prepare` in npm@5; frankly speaking, I don't know the best practice how to deal with it, other than ignoring or `sed`ing `package.json` in the `before_install` hook:

```yaml
before_install:
- 'if [[ "$TRAVIS_NODE_VERSION" == 8 ]]; then sed -i -r \'s/"prepublish": "(.*)"/"prepare": "\1"/\'; fi'
```